crates/circuit/src/dag_circuit.rs

```diff
@@ -4517,6 +4532,21 @@ impl DAGCircuit {
             .len()
     }
 
+    #[getter]
```

Comment:
I would add a comment here to say these are for testing:

```suggestion
    // These three stretch getter methods are for testing purposes, they'll be public
    // APIs in Qiskit 2.1
    #[getter]
```